ceph/rados/cephvolume_workflows.py

```diff
@@ -0,0 +1,387 @@
+            osd_fsid: FSID of OSD to activate
+            all: activate all OSD that are prepared for activation by passing --all flag
+        Returns:
+            Returns the output of ceph-volume lvm activate
+        """
+        _cmd = "ceph-volume lvm activate"
+        if all:
+            _cmd = f"{_cmd} --all"
+        else:
+            _cmd = f"{_cmd} --bluestore {osd_id} {osd_fsid}"
+        return self.run_cv_command(cmd=_cmd, host=host)
+
+    def lvm_deactivate(self, osd_id: int, host):
+        """Module to remove the volume groups and the logical volume
+        Args:
+            None
+        Returns:
+            Returns the output of ceph-volume lvm deactivate
+        """
+        _cmd = f"ceph-volume lvm deactivate {osd_id}"
+        return self.run_cv_command(cmd=_cmd, host=host)
+
+    def lvm_batch(self, devices: list, host):
+        """Module to automate the creation of multiple OSDs
+        Args:
+            devices: list of devices for OSD creation
+        Returns:
+            Returns the output of ceph-volume lvm batch
+        """
+        _cmd = "ceph-volume lvm batch --bluestore"
+        for device in devices:
+            _cmd = f"{_cmd} {device}"
+        return self.run_cv_command(cmd=_cmd, host=host)
+
+    def lvm_migrate(
+        self, osd_id: int, osd_fsid: str, host, from_flag_list: list, target: str
+    ):
+        """Module to migrate the BlueStore file system (BlueFS) data (RocksDB data)
+        from the source volume to the target volume.
+        Args:
+            from: arguement for --from flag. list can contain "data" "db" "wal"
+            osd_id: ID of the source OSD for data migration
+            osd_fsid: FSID of the source OSD for data migration
+            target: target device/logical volume for data migration
+        Returns:
+            Returns the output of ceph-volume lvm migrate
+        """
+        _cmd = f"ceph-volume lvm migrate --osd-id {osd_id} --osd-fsid {osd_fsid} --from"
+        for device in from_flag_list:
+            _cmd = f"{_cmd} {device}"
+        return self.run_cv_command(cmd=_cmd, host=host)
+
+    def lvm_create(self, path: str, host):
+        """Module to call the prepare subcommand, and then calls the activate subcommand.
+        Args:
+            path: path can be device path (/dev/vdb) or logical volume (volume_group/logical_volume)
+        Returns:
+            Returns the output of ceph-volume lvm create
+        """
+        _cmd = f"ceph-volume lvm create --bluestore --data {path}"
+        return self.run_cv_command(cmd=_cmd, host=host)
+
+    def lvm_new_db(self, path: str, host):
+        """Module to attache the given logical volume to the given OSD as a DB volume
+        Args:
+            path: path can be device path (/dev/vdb) or logical volume (volume_group/logical_volume)
+        Returns:
+            Returns the output of ceph-volume lvm new-db
+        """
+        _cmd = f"ceph-volume lvm create --bluestore --data {path}"
```

Comment:
command used here is same as method `def lvm_create()`

Reply:
Missed changing the command. Updated functions :- https://github.com/red-hat-storage/cephci/blob/780718b450dcba550afcc5dc98cd81f5ef20d2fd/ceph/rados/cephvolume_workflows.py#L240C1-L268C56 

`
_cmd = f"ceph-volume lvm new-db --osd-id {osd_id} --osd-fsid {osd_fsid} --target {volume_group}/{logical_volume}"
`